### DISCUSSION TITLE
about-me

### DISCUSSION THREAD

**Discussion by giscus** [MrPowerGamerBR/PersonalWebsiteComments#2] [Fan Comments]
# about-me

Howdy! Eu me chamo MrPowerGamerBR (Leonardo), tenho 22 anos e você chegou na minha casa na internet! Se sinta em casa e não repare a bagunça.

https://mrpowergamerbr.com/br/about-me/

**Comment by MrPowerGamerBR** [comment]
owo whats this?? :3

**Reply by modulador592** [reply]
owo

**Reply by yxqsnz** [reply]
uwu

**Reply by WinG4merBR** [reply]
owo

**Reply by dorpew** [reply]
uwu :3

**Comment by http-pixelon** [comment]
>.>

**Reply by http-pixelon** [reply]
Oxi... era pra ser isso:  >.>

**Reply by MrPowerGamerBR** [reply]
é que quando coloca `>` no GitHub (e no Discord), ele cria como se fosse uma citação

**Comment by modulador592** [comment]
Porra hein não paro de receber eemaijl sobre isso
De: ***@***.***>
Enviado:quinta-feira, 12 de maio de 2022 13:42
Para: ***@***.***>
***@***.***>; ***@***.***>
Assunto: Re: [MrPowerGamerBR/PersonalWebsiteComments] about-me (Discussion #2)


é que quando coloca > no GitHub (e no Discord), ele cria como se fosse uma citação

**Reply by WinG4merBR** [reply]
![image](https://user-images.githubusercontent.com/68250074/168126571-3c8ebb31-6f82-4ee1-b9a5-e872246ca956.png)

**Comment by Kiwiie** [comment]
(✿◡‿◡)

**Comment by oluizfcs** [comment]
Admiro muito o seu trabalho. Futuramente pretendo cursar ciência da computação e ler isso me motivou ainda mais.

**Reply by MrPowerGamerBR** [reply]
:3

**Comment by modulador592** [comment]
Tambem pretendo cursar ciencia da tecnologia :3

Em qua., 28 de dez. de 2022 às 00:43, MrPowerGamerBR <
***@***.***> escreveu:

**Comment by cookieukw** [comment]
Muito bom. Eu tenho uma chatbot desde 2019 e quero fazer uma rede neural. No início foi só uma alternativa ao simsimi mas quero melhorá-la ainda mais. Acabei achando esse site pesquisando sobre algum dataset do simsimi

**Comment by ari-rocha** [comment]
Obrigado por ser uma das pessoas que me inspiraram a entrar de cabeça na programação 🙃

**Comment by BrunoFelixDeSousa** [comment]
mrpowergamerbr, qual dica você pode me passar para chegar nesse nível com kotlin? tem algum material que você recomenda?

**Comment by silverzin8** [comment]
Brabo dms :)
